### DISCUSSION TITLE
Tools or Plug-in for JSON-LD explanation

### DISCUSSION THREAD

**Discussion by hsinpeng** [LinkedSoftwareDependencies/Components.js#89] [Q&A] [answered]
It's difficult to see the whole picture by reading a bunch of config files. For example, understand how CSS works by importing "file.json" configuration.
Since config files are written in JSON-LD format, is any tool or vscode plugi-n can help me to understand the relationship of components more easily? Thanks.

**Comment by joachimvh** [comment, marked as answer]
Unfortunately there is no such tooling (I would also be a fan 😄). The only thing I can think of is loading all the JSON-LD files into an RDF graph visualization to get an overview but I'm not sure how well that would work.

**Reply by hsinpeng** [reply]
Got it. Thanks.
